day2.md

```diff
@@ -0,0 +1,171 @@
+## Day2 Tues. 09/06/16 [Tokyo 東京]
+---
+
+10:00 - 11:00 [Tsukiji Fish Market](#fish_market)<br />
+12:00 - 14:20 [Tokyo Skytree Town](#skytree)<br />
+15:00 - 16:00 [Sensō-ji](#temple)<br />
+17:00 - 19:00 [Shinjuku & Fuunji Ramen(風雲児)](#shinjuku)<br />
+20:00         [Department Store](#store)
+
+---
+There was no better way to start our second day in Japan than going to <a name="fish_market">the Tsukiji fish market</a>!
+> "Tsukiji Market (築地市場, Tsukiji Shijō) is a large wholesale market for fish, fruits and vegetables in central Tokyo. It is the most famous of over ten wholesale markets that handle the distribution of fish, meat, produce and flowers in Tokyo. Tsukiji Market is best known as one of the world's largest fish markets, handling over 2,000 tons of marine products per day." -[Japen Guide](http://www.japan-guide.com/e/e3021.html)
+
+Many tourists will come here by 5 a.m. for the [live tuna auctions](https://www.youtube.com/watch?v=uv5DloVT3iU). Knowing sleep is the second most important thing, after food, in our lives, we saved ourselves the troubles of thinking wishfully, setting a 4 am alarm, turning it off as soon as it rings, and going right back to sleep. Instead, after a great night of sleep, we woke up naturally around 9 am and headed straight to the Tsukiji fish market with only one clear goal in mind: food!
+
+<img src="day2/14435132_1115751911807835_676936538936024301_o.jpg" width="500"><br/><br/>
+<img src="day2/IMG_20160906_101226.jpg" width="500"><br/>
+
+Walking around the market and surrounding by foods, especially fresh seafoods, got me very excited. The 5-years-old inside me had been screaming, jumping, and dancing all around the whole time as we were walking through the main street and deciding where we should have our first meal of the day. Many of you should be familiar with the analogy that life sometimes is like walking through a forest knowing that you can only pick one fruit from a tree without knowing what's ahead of you nor the opportunity of returning and picking from a tree you have passed. That's exactly how I felt when we were in the fish market and any other times we had to make a decision related to food in Japan! We could only ate so much each day and there were so many yummies surrounding and calling us. As a computer science major, I couldn't help but tried to collect as many relevant data as possible and run it through my real-time "where-should-we-eat" algorithm<sup>[1](#algorithm)</sup>.
```

Comment:
- "**being surrounded** by **food**" instead of "surrounding by foods"
 - See [this stackexchange question](http://english.stackexchange.com/questions/58324/food-vs-foods-spelling-and-meaning) on "foods" vs "food"
- "fresh **seafood**" instead of "fresh seafood"
- "5-**year**-old" instead of "5-years-old"
- "**was** screaming" instead of "had been screaming"
 - "Had been" would imply the [pluperfect tense](https://en.wiktionary.org/wiki/pluperfect_tense), where the verb is happening before a past event. Here the verb happening _while_ you are walking, but not _before_ any event, so the pluperfect is not as appropriate.)
- "could only **eat**" instead of "could only eat"
 - "Could eat" is the same in the past and present because of the "could"
- Add a comma to separate two independent clauses: "...each day**,** and there were..."
- "couldn't help but **try**" instead of "couldn't help but tried"
 - Same thing as above with "could"
- "as **much** relevant data" instead of "as many relevant data"
 - "Many" is used for countable things (like data **points** or apples) whereas "much" is used for noncountable things (like hunger).